### DISCUSSION TITLE
To add the wonderfull versatille_thermostat into Energy

### DISCUSSION THREAD

**Discussion by lio908** [jmcollin78/versatile_thermostat#219] [Ideas]
Hello to all
Is it possible to ad feedback of the consumation to the Energy?

**Comment by jmcollin78** [comment]
Yes. It is already possible though the energy entity linked to the VTherm.

**Comment by lio908** [comment]
Bonjour
Yes...Yes?
En fait, c'est possible ou c'est une bonne idée?
Dans la mesure où on peut mettre la puissance du radiateur par thermostat,
avec la durée de pilotage, serait-il possible de remonter cette
consommation côté énergie, consommation réseau?
?

Le mer. 22 nov. 2023 à 13:10, Jean-Marc Collin ***@***.***> a
écrit :

**Reply by jmcollin78** [reply]
Oui c'est déjà possible, ca sert même exactement à ça. Le mieux c'est d'essayer comme ça vous avez tout de suite la réponse.

**Comment by lio908** [comment]
Apparemment, il y a un filtrage sur le type énergie quand on cherche les
paramètres à prendre en compte.
Que dois-je mettre, en jaune?

[image: image.png]

Le mer. 22 nov. 2023 à 14:20, Jean-Marc Collin ***@***.***> a
écrit :

**Reply by jmcollin78** [reply]
Y a pas l'image

**Comment by lio908** [comment]
Je retente
[image: image.png]


Le mer. 22 nov. 2023 à 14:37, Jean-Marc Collin ***@***.***> a
écrit :

**Comment by jmcollin78** [comment]
Toujours pas. Tu la vois de ton coté l'image ?

**Comment by lio908** [comment]
Oui, j'utilise outils capture de W10 et je colle
Sinon c'est quoi comme info que energy peut prendre comme variable entrée
Quand je fait ajouter, il n'y a rien qui provienne de Versatile Ther.
C'est pour ça que je demande d'en créer une.
Sinon cela veut dire que HA filtre l'insertion par les info type energy
C'est ce que je voie quand j'insère une variable, que celle finisant par
energy, par ex celles qui proviennent de mes Shilly


Le mer. 22 nov. 2023 à 15:20, Jean-Marc Collin ***@***.***> a
écrit :

**Reply by jmcollin78** [reply]
C'est l'entité Energy :
![Capture d’écran 2023-11-22 à 15 55 21](https://github.com/jmcollin78/versatile_thermostat/assets/1712374/d6f58b99-9dc8-492b-a346-a614fc0401ce)


Pour la voir il faut qu'elle soit historisée et attendre un peu (cf. recorder de HA). C'est peut être ca ton soucis

**Comment by lio908** [comment]
Ha
et donc quand je crée un Vers.Therm, j'ai cette info energy?

ici, je la trouve pas
https://github.com/jmcollin78/versatile_thermostat
Parameters synthesis
il y a un truc que je comprends pas...?

Le mer. 22 nov. 2023 à 15:57, Jean-Marc Collin ***@***.***> a
écrit :

**Reply by jmcollin78** [reply]
Il faut que la gestion de la puissance soit activée sur le VTherm et tu le verras dans les attributs du VTherm:
```
...
device_power: 1
mean_cycle_power: null
total_energy: 0
```
Ca s'appelle comme ça (dans les attributs du VTherm).

**Comment by lio908** [comment]
MERCI BEAUCOUP !
j'avais les info...

Le mer. 22 nov. 2023 à 17:01, Jean-Marc Collin ***@***.***> a
écrit :

**Comment by lio908** [comment]
Bonjour
Comment tester que le VTherm. est "vivant " ou mort, cad inactif, planté
quoi.
J'ai eu un VT Therm qui ne gérait plus l'entrée de la temp et comme elle
était en dessous de la ref, la sortie était tj active. La pièce à été
chauffée en permanence.
Y a-t-il une solution pour parer à ce pb?


Le mer. 22 nov. 2023 à 19:52, lio cies ***@***.***> a
écrit :

**Reply by jmcollin78** [reply]
Un VTherm n'est jamais planté (ou disons qu'il ne devrait pas mais j'ai pas d'exemples de VTherm planté).
Le truc qui pourrait ressembler c'est le mode sécurité qui s'active lorsque le VTherm ne reçoit plus de temperature de la part du capteur de température intérieur ou extérieur. Dans ce cas là, il refuse de chauffer pour éviter l'incendit mais garde un 10% de chauffe.

**Reply by jmcollin78** [reply]
Si il a un problème tu peux aller voir dans les logs (journal) si tu vois des erreurs dues au VTherm.

**Comment by lio908** [comment]
Ex:
 [image: image.png]

Vers 13hr, plantage et la sortie en orange o activé le chauffage, vers 17hr
30, quand j'ai "rebooté" le système, on voit la température qui était à 20°

Du coup ou est la sécurité?

Le mar. 5 déc. 2023 à 19:29, Jean-Marc Collin ***@***.***> a
écrit :

**Reply by jmcollin78** [reply]
Y a toujours pas d'image... je peux pas t'aider.

**Comment by lio908** [comment]
je regarde les log...

Le mar. 5 déc. 2023 à 20:08, lio cies ***@***.***> a
écrit :

**Comment by lio908** [comment]
Dans mon log, la sortie était à 1 donc chauffage, puis plus info
jusqu'a que je rentre chez moi. HA était planté j'ai du le rebouter
plusieur fois.
C'est le pb , si HA plante, les sorties qui étaient activées restent dans
état...


Le mar. 5 déc. 2023 à 20:10, Jean-Marc Collin ***@***.***> a
écrit :

**Comment by lio908** [comment]
Cependant, il m'est arrivé aussi un autre pb, la sonde de temp n'était plus
rafraîchie et comme en dessous de la consigne, le chauffage était à on tout
le temps. Il n'y a pas de sécu qui peut vérifier si l'entrée est "vivante"?


Le mar. 5 déc. 2023 à 20:18, lio cies ***@***.***> a
écrit :

**Comment by jmcollin78** [comment]
> Il n'y a pas de sécu qui peut vérifier si l'entrée est "vivante"?

Si justement et ca s'appelle le mode de sécurité. Si il n'y a plus de température le thermostat se met en sécurité.

> HA était planté j'ai du le rebouter plusieur fois.

Si ton HA a des problèmes, il faut d'abord que tu résolves ces problèmes avant d'essayer de faire marcher le VTherm.

Ecoute, je te propose d'arrêter là, tu ne fais pas beaucoup d'effort pour que je puisse t'aider (pas d'image, pas d'attributs du VTherm, explications franchement pas claires) et si tu as des problèmes par ailleurs sur HA, c'est par là qu'il faut commencer.

**Comment by lio908** [comment]
Merci quand même, pour l'instant, ce "climate" est le meilleur que j'ai eu.
Pour les attributs des VTherm que je crées, dans quel fichier peut-on les
recup?

*Si justement et ca s'appelle le mode de sécurité. Si il n'y a plus de
température le thermostat se met en sécurité.*
==> ben apparement non, car si la sonde de temp est "figée" sans évolution
en + ou en -, la sortie est tj pilotée.

Pour les pb de HA, c'est le grand mystère de HA, parfois ça coince et puis
ça revient tout seul, sans rien faire...
==>OK c'est hors sujet pour le VT, mais ça le dédouane, il n'est pas en
tort...

Dernière remarque, je joint un PDF sur lequel j'ai collé 1 print screen
C'est 2 thermostats donc, sur la chambre de Mathias, on voit que si la tem
est dépassé, les sorties sont arrêtées, sur celle de Ludo, c'est pas le cas.
Les paramètres sont identiques. La temp dépasse de beaucoup, c'est normal
car j'ai allumé la cheminée...
Par contre pourquoi sur un therm, la sortie est encore activée, mystère...

Merci de votre aide.

Le mar. 5 déc. 2023 à 21:07, Jean-Marc Collin ***@***.***> a
écrit :
